cvmfs/server/cvmfs_server_publish.sh

```diff
@@ -206,8 +206,14 @@ cvmfs_server_publish() {
     if [ "x$CVMFS_UNION_FS_TYPE" != "x" ]; then
       sync_command="$sync_command -f $CVMFS_UNION_FS_TYPE"
     fi
-    if [ "x$CVMFS_GENERATE_LEGACY_BULK_CHUNKS" = "xtrue" ]; then
-      sync_command="$sync_command -O"
+    if [ "x$CVMFS_GENERATE_LEGACY_BULK_CHUNKS" != "x" ]; then
+      if [ "x$CVMFS_GENERATE_LEGACY_BULK_CHUNKS" = "xtrue" ]; then
+        sync_command="$sync_command -O"
+      fi
+    else
+      if [ "x$CVMFS_DEFAULT_GENERATE_LEGACY_BULK_CHUNKS" = "xtrue" ]; then
+        sync_command="$sync_command -O"
+      fi
```

Comment:
A simpler way to write this would be
```
if [ "x${CVMFS_GENERATE_LEGACY_BULK_CHUNKS:-$CVMFS_DEFAULT_GENERATE_LEGACY_BULK_CHUNKS}" = "xtrue" ]; then
  sync_command="$sync_command -O"
fi
```